Realm/Tests/RealmSwift/SwiftArrayTests.swift

```diff
@@ -0,0 +1,143 @@
+////////////////////////////////////////////////////////////////////////////
```

Comment:
Delete these old test files? Everything in them should now be covered by better tests.